src/main/java/uk/gov/hmcts/darts/notification/service/impl/NotificationServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.darts.notification.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.darts.notification.dto.CreateNotificationReq;
+import uk.gov.hmcts.darts.notification.entity.Notification;
+import uk.gov.hmcts.darts.notification.enums.NotificationStatus;
+import uk.gov.hmcts.darts.notification.repository.NotificationRepository;
+import uk.gov.hmcts.darts.notification.service.NotificationService;
+
+import java.sql.Timestamp;
+
+
+@RequiredArgsConstructor
+@Service
+public class NotificationServiceImpl implements NotificationService {
+
+    @Autowired
```

Comment:
This `@Autowired` can be dropped here if we are using constructor arg instantiation using the lombok annotation

Reply:
Not quite true as notificationRepo isn't marked as final, but i can do.

Reply:
have added final and removed @Autowired